### PR TITLE
Fix no docker-images load before e2e test

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -62,6 +62,11 @@ make test-integration
 Run e2e testing. During e2e testing, hub and managed cluster share the same Kubernetes cluster.
 ```shell
 export KUBECONFIG=</path/to/kubeconfig>
+
```
Comment (qiujian16) [inline CONTRIBUTING.md:67]: should we also add go install image-builder here?

Comment (xuezhaojun) [reply CONTRIBUTING.md:67]: Of course. 
Added.

**Pull request merged**